index.html

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <h1>Web Bluetooth Demos</h1>
 <ul>
-  <li><a href="https://webbluetoothcg.github.io/demos/bluetooth-toy-plane/">bluetooth-toy-plane</a></li>
```

Comment:
Let's remove this part from this PR for now as it's only about Web Bluetooth API not being available.